templates/transcripts-gene-pc-rev.yaml

```diff
@@ -0,0 +1,21 @@
+genome_id: <genome_id>
+category:
+  track_category_id: genes-transcripts
+  label: Genes & transcripts
+  type: Genomic
+trigger:
+  - track
+  - gene-pc-rev
+type: gene
+label: Protein coding genes
+additional_info: Reverse strand
+colour: DARK_GREY
+display_order: 100
+on_by_default: true
+description: Shows all protein coding genes on the reverse strand. <annotation_description>
+sources:
+  - name: <annotation_method>
+    url: <annotation_url>
+datafiles:
+  gene: transcripts.bb
```

Comment:
github complains about this file; possibly because the next line is both empty and indented:

<img width="240" alt="image" src="https://github.com/user-attachments/assets/0ff37920-33c7-4f5f-a811-747131b88fbf">

this is different from all other yaml templates; probably makes sense to clean this up